### DISCUSSION TITLE
Sequelize create method returns null value but in my table  is't correctly created

### DISCUSSION THREAD

**Discussion by GopalPatidar54** [sequelize/sequelize#15507] [Help & Questions] [answered]
Sequelize create method returns a null value on those fields which are created using the before_insert trigger. Anyway, create a method to return the correct value.

I create a table of users {id, name, last_name, uuid}

create a Row-Level Trigger before_insert on uuid
before_insert_users | INSERT | users | BEGIN SET new.uuid = uuid();

user.create({name:"cyz",last_name:"abc"}), return value {id:213,name:"cyz",last_name:"abc" ,uuid:null }

select * from users;
{id:213,name:"cyz",last_name:"abc" ,uuid: "98ba5d71-85e5-11ed-aa69-b06088d9a387" };

there is any way to return the correct uuid in create method response?

**Comment by ephys** [comment, marked as answer]
If you're using a database that doesn't support `RETURNING` on `INSERT` queries, then values generated by the database won't be available until you can `.reload` on your model instance

**Reply by GopalPatidar54** [reply]
I'm using MySQL database

**Reply by ephys** [reply]
MySQL doesn't support `INSERT RETURNING`

**Reply by GopalPatidar54** [reply]
so is there any option to return the correct UUID in create method response?

**Reply by ephys** [reply]
You need to run a separate SELECT query to get it, which you can do by calling [the reload method](https://sequelize.org/docs/v6/core-concepts/model-instances/#reloading-an-instance)
